ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -240,6 +240,13 @@
   @Default("2")
   public final long storeStatsRecentEntryProcessingIntervalInMinutes;
 
+  /**
+   * Period in minutes to specify how frequent the validDataSizeCollector executed.
+   */
+  @Config("store.get.valid.size.interval.in.minutes")
+  @Default("2")
+  public final long storeGetValidSizeIntervalInMinutes;
```

Comment:
Change its time unit to second in case we want to have a interval less than 1 minute?

Reply:
Done